### PR TITLE
CSS style guide updates for md

### PR DESCRIPTION
**Pull request by wbamberg**
This updates the [CSS style guide](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/CSS_style_guide), and the changelog, to make it in line with writing "MDN-Markdown"-compatible HTML.

* `.hidden`, `.example-good`, `.example-bad` should *only* be used on code blocks
* `.seoSummary` should not be used
* `<details>` elements should not be used
* `.standard-table` isn't relevant any more

Given those changes I thought a bit of restructuring of the guide made sense. Also, some headings contained code formatting, which is against our style guide, so I removed it.

There are lots of changes but I made them in separate commits, which I hope helps review.

### CODE REVIEW

File: files/en-us/mdn/contribute/changelog/index.html [hunk 1]
```diff
@@ -12,11 +12,29 @@
...
-
-<p>See our <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard">Guide to classes and styles used in MDN content</a> guide for further information and syntax guides.</p>
+<h3>Updates to CSS style guide for Markdown</h3>
+
+<p>Multiple updates to the <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide">CSS style guide</a> to reflect the move towards Markdown, and encourage authors to write HTML in a Markdown-compatible way.</p>
+
+<ul>
+  <li>
+    <p><a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard">Note</a> and <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.notecard.warning">warning</a> boxes no longer have a separate <code>&lt;h4&gt;</code> heading for the title (e.g. <code>&lt;h4&gt;Warning&lt;/h4&gt;</code> ). Instead, the the first paragraph begins with title text, as shown: <code>&lt;strong&gt;Note:&lt;/strong&gt; </code> and <code>&lt;strong&gt;Warning:&lt;/strong&gt; </code>. Note that a space character is required after the closing <code>&lt;/strong&gt;</code> tag/before the content).</p>
```
Comment (teoli2003) [inline files/en-us/mdn/contribute/changelog/index.html:21]: A duplicate `the`:
```suggestion
    <p><a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.note.notecard">Note</a> and <a href="/en-US/docs/MDN/Guidelines/CSS_style_guide#.notecard.warning">warning</a> boxes no longer have a separate <code>&lt;h4&gt;</code> heading for the title (e.g. <code>&lt;h4&gt;Warning&lt;/h4&gt;</code> ). Instead, the first paragraph begins with title text, as shown: <code>&lt;strong&gt;Note:&lt;/strong&gt; </code> and <code>&lt;strong&gt;Warning:&lt;/strong&gt; </code>. Note that a space character is required after the closing <code>&lt;/strong&gt;</code> tag/before the content).</p>
```
(Not me making the approval, though)

Comment (wbamberg) [reply files/en-us/mdn/contribute/changelog/index.html]: thanks!

**Review by rachelandrew — APPROVED** [review]
thanks - this all looks good from my pov :)

**Comment by hamishwillee** [comment]
@wbamberg Awesome.

Do we need to somehow capture "cleanup" and ways of working now? The specific case I'm thinking of is seosummary.
- Do we need to say somewhere that if you see a span with SEO summary it should be removed?
- I do think (perhaps in https://developer.mozilla.org/en-US/docs/MDN/Contribute/Markdown_in_MDN) we should note that the first non-macro paragraph in a page is used as a summary, to help people understand this should be succinct.

Hope that makes sense. Either way "this" change is good.

**Comment by wbamberg** [comment]
> @wbamberg Awesome.
> 
> Do we need to somehow capture "cleanup" and ways of working now? The specific case I'm thinking of is seosummary.
> 
>     * Do we need to say somewhere that if you see a span with SEO summary it should be removed?

We could. I mean there's lots of cleanup we need to do, and so far I'm just organizing this as I plan for converting each section. It might be worth writing a general guide to "things you can do to help us convert", but it might be better just to file issues for each thing.

>     * I do think (perhaps in https://developer.mozilla.org/en-US/docs/MDN/Contribute/Markdown_in_MDN) we should note that the first non-macro paragraph in a page is used as a summary, to help people understand this should be succinct.

But we should definitely do this, yes, as there are clearly implications for writers here.

> 
> Hope that makes sense. Either way "this" change is good.

👍

**Pull request merged**